rita/src/client.rs

```diff
@@ -77,6 +77,8 @@ use rita_client::dashboard::network_endpoints::*;
 use rita_common::dashboard::network_endpoints::*;
 use rita_common::network_endpoints::*;
 
+extern crate openssl_probe;
```

Comment:
Put this along with other `extern crate`s. Our imports look like garbage in many places, but let's not make it worse.